_sass/_layout.scss

```diff
@@ -219,13 +219,23 @@
 .post-list {
   margin-left: 0;
   list-style: none;
+  display: flex;
```

Comment:
oh huh I probably would have done it in grid but cool to see that you can do a grid layout in flexbox also!